presentation/src/main/java/com/globant/equattrocchio/cleanarchitecture/mvp/view/ImageDialogFragment.java

```diff
@@ -0,0 +1,69 @@
+package com.globant.equattrocchio.cleanarchitecture.mvp.view;
+
+import android.app.Dialog;
+import android.app.DialogFragment;
+import android.content.DialogInterface;
+import android.os.Bundle;
+import android.support.v7.app.AlertDialog;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.bumptech.glide.Glide;
+import com.globant.equattrocchio.cleanarchitecture.R;
+import com.globant.equattrocchio.data.response.Image;
+import com.google.gson.Gson;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+public class ImageDialogFragment extends DialogFragment {
+
+    @BindView(R.id.image_detail)
```

Comment:
put all these variables like this:
`@BindView(R.id.text_url) TextView textUrl;`